### PR TITLE
Fixes #467 : Now the reminder can be set from the app itself. 

### LINKED ISSUE #467: Setting reminder through the susi app itself.
**Actual Behaviour**

At present , we cannot set the reminder from susi.

**Expected Behaviour**

Susi must be intelligent enough to set the reminder on users demand like google allo.

**Would you like to work on the issue?**

Yes , I am already working on it.

### PR DESCRIPTION
**Pull request by mayank408**
Fixes issue #467 

Changes: Now reminder can be set from the app itself.

Screenshots for the change: 

unknown
unknown2

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java [hunk 1]
```diff
@@ -460,7 +467,7 @@ public void onRequestPermissionsResult(int requestCode, String permissions[], in
                 break;
             }
         }
-    }
```
Comment (Abhi2424shek) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: The indentation seems to be a bit wrong.

File: app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java [hunk 2]
```diff
@@ -650,9 +657,11 @@ private void sendMessage(String query) {
     private synchronized void computeOtherMessage() {
         final String query;
         final long id;
-        String answer_call=null;
+        String answerCall = null;
         String google_search = null;
+        String reminder = null;
         String setAlarm = null;
+        final String[] reminder_date = {null};
```
Comment (Abhi2424shek) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: Pls use camelCase

File: app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java [hunk 3]
```diff
@@ -788,6 +806,41 @@ public void execute(Realm bgRealm) {
...
+                                                case 0:{
+                                                    MainActivity.this.reminder = query;
+                                                    reminder_date[0] = getString(R.string.reminder_date);
+                                                    answer = reminder_date[0];
+                                                    isWebSearch = false;
+                                                    Log.d(TAG, "onResponse: " + MainActivity.this.reminder);
+                                                    reminderQuery =1;
+                                                    break;
+                                                    }
+                                                case 1:{
```
Comment (Abhi2424shek) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: Is it necessary to apply scoping?

**Comment by mayank408** [comment]
@the-dagger @daminisatya @jig08 @mananwason could you pls review this...

**Comment by daminisatya** [comment]
Squash your commits.

**Comment by Abhi2424shek** [comment]
We could use squash and merge.

**Comment by daminisatya** [comment]
@Abhi2424shek Read the rules 

https://github.com/fossasia/susi_android#code-practices

**Comment by mayank408** [comment]
@daminisatya Commits merged.

**Comment by daminisatya** [comment]
great!

**Pull request merged**